proto/cv/scanner/v1/scanner.proto

```diff
@@ -0,0 +1,84 @@
+syntax = "proto3";
+
+package cv.scanner.v1;
+
+import "google/api/annotations.proto";
+import "google/api/http.proto";
+import "ssn/annotator/v1/annotator.proto";
+import "ssn/type/candidate.proto";
+
+option go_package = "github.com/e-conomic/vmlapis/gen/go/cv/cv/v1;cv";
+
+service CvScanner {
```

Comment:
It feels a little off when service is `CvScanner` but everything else is `CV...`, but I don't care as much if it's bother to change 😄

Reply:
I will leave it as Scanner with scanCV method